src/anm/player.js

```diff
@@ -385,6 +386,7 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
                     if (player.anim === result) { // avoid race condition when there were two requests
                         // to load different animations and first one finished loading
                         // after the second one
+                        player.playedOnce = false;
```

Comment:
good point, I had some special reason but since I cannot recall details, it's invalid. Anyway, this flag was intended to be used for end-screen, which I had no chance to finish, so I will just clean things up.